Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -438,7 +438,13 @@ def run(self):
 class ArchiveMinifiedBuiltProduct(ArchiveBuiltProduct):
     name = 'archive-minified-built-product'
     command = ["python3", "Tools/CISupport/built-product-archive",
-               WithProperties("--platform=%(fullPlatform)s"), WithProperties("--%(configuration)s"), "archive", "--minify"]
+               WithProperties("--platform=%(fullPlatform)s"), WithProperties("--%(configuration)s")]
+
+    def run(self):
+        self.appendCrossTargetFlags(self.getProperty('additionalArguments'))
+        self.command.append('archive')
+        self.command.append('--minify')
+        return super().run()
```

Comment:
There's no need to do this; we can just leave the `--minify` in the command and use the parent class `run` method.